testing/py-pkginfo/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: William Waisse <alpine@ww7.be>
+# Maintainer: William Waisse <alpine@ww7.be>
+pkgname=py-pkginfo
+_pkgname=pkginfo
+pkgver=1.4.2
+pkgrel=2
+pkgdesc="This package provides an API for querying the distutils metadata written in the PKG-INFO file inside a source distriubtion"
+url="https://pypi.org/project/pkginfo/"
+arch="noarch"
+license="Apache-2.0"
+depends=""
+makedepends="python2-dev python3-dev py-setuptools py3-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+#source="https://files.pythonhosted.org/packages/39/c9/111ececbeed8e69cd1b6bec79a32a0b0f6074038a4244e58e285ad278248/pkginfo-1.4.2.tar.gz"
```

Comment:
needs fix and also why no `check()` implemented?

Reply:
I have no idea what test I should put in check,or where to find them. 
having a commented line with # is a problem ?